aspnetcore-features/HttpContextItems/Tests/NoiseMakerUnitTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using NoiseMakerNames;
+
+namespace Tests
+{
+    public class NoiseMakerUnitTest
+    {
+        [Fact]
+        public void WhenAddNoiseMakerIsCalled_ThenNewNoiseMakerIsAddedToTheList()
+        {
+            //arrange
+            NamesOfNoiseMakers namesOfNoiseMakers = new();
+            NoiseMaker noiseMaker = new()
+            {
+                Name = "Mira",
+                Severity = 2,
+                FirstAppearance = true
+            };
+
+            //act
+            namesOfNoiseMakers.AddNoiseMaker(noiseMaker);
+            List<NoiseMaker> noiseMakerList = namesOfNoiseMakers.GetNoiseMakers();
+
+            //assert
+            Assert.NotNull(noiseMakerList);
+            Assert.Contains(noiseMaker, noiseMakerList);
```

Comment:
Please use vars where you can. You can also use new() but we prefer var since we use it across the codebase.